src/Peachpie.Runtime/Reflection/PhpRoutineInfo.cs

```diff
@@ -335,6 +363,8 @@ static bool AllDeclaredInBase(MethodInfo[] methods, Type callertype)
 
         public override PhpTypeInfo DeclaringType => _methods[0].DeclaringType.GetPhpTypeInfo();
 
+        public override IPhpCallable BindTarget(object target) => new BoundTargetCallable(target, PhpInvokable);
```

Comment:
remove

Reply:
OK, updated in 2828173f318c841df8a368d1e4817d6ec702e2b6.